.github/actions/publish_release/action.yml

```diff
@@ -27,11 +27,11 @@ runs:
       with:
         enable_docker_multibuild: true
     - uses: ./.github/actions/restore_artifacts
-    - run: promu crossbuild tarballs
+    - run: ~/go/bin/promu crossbuild tarballs
       shell: bash
-    - run: promu checksum .tarballs
+    - run: ~/go/bin/promu checksum .tarballs
       shell: bash
-    - run: promu release .tarballs
+    - run: ~/go/bin/promu release .tarballs
```

Comment:
This attempted fix comes from [here](https://github.com/prometheus/prometheus/blob/8dbb2eaf0bafd3563dc7d295fefa47cbb8c2a36a/.github/actions/build/action.yml#L19) which seems to be working

Reply:
We will know if this change works when I push another release tag for this PR